docs/monorail/ubuntu_install_instructions.rst

```diff
@@ -60,80 +60,39 @@ Set the Links to the Mirrors
 
   .. code::
 
-    sudo ln -s /var/mirrors/esxi /var/renasar/renasar-http/static/http/vmware
-    sudo ln -s /var/mirrors/esxi /var/renasar/renasar-tftp/static/tftp/vmware
-    sudo ln -s /var/mirrors/ubuntu_trusty /var/renasar/renasar-http/static/http/ubuntu_trusty
-    sudo ln -s /var/mirrors/ubuntu/14.04/mirror/mirror.pnl.gov/ubuntu/ /var/renasar/renasar-http/static/http/trusty
-    sudo ln -s /var/mirrors/centos /var/renasar/renasar-http/static/http/centos
-    sudo ln -s /var/mirrors/suse /var/renasar/renasar-http/static/http/suse
+    sudo ln -s /var/mirrors/esxi <on-http directory>/static/http/vmware
+    sudo ln -s /var/mirrors/esxi <on-tftp directory>/static/tftp/vmware
+    sudo ln -s /var/mirrors/ubuntu_trusty <on-http directory>/static/http/ubuntu_trusty
+    sudo ln -s /var/mirrors/ubuntu/14.04/mirror/mirror.pnl.gov/ubuntu/ <on-http directory>/static/http/trusty
+    sudo ln -s /var/mirrors/centos <on-http directory>/static/http/centos
+    sudo ln -s /var/mirrors/suse <on-http directory>/static/http/suse
 
 Install supplementary files:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
   .. code::
 
-    sudo apt-get install renasar-static-common renasar-mibs
-    sudo apt-get install renasar-static-coreos
-    sudo apt-get install renasar-static-emc
-    sudo apt-get install renasar-static-arista
-    sudo apt-get install renasar-static-intel
-    sudo apt-get install renasar-static-vmware
-    sudo apt-get install renasar-static-xen
+    sudo apt-get install on-static-common
+
+  TODO: add more instructions here
 
 Retrieve the Installer/Distribution Media
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  .. code::
-
-    mkdir -p /var/mirrors
-    cd /var/mirrors
-    curl http://xfer.renasar.com/emc/ubuntu_trusty.tgz | tar xvzf -
-    curl http://xfer.renasar.com/emc/esxi.tgz | tar xvzf -
+  TODO: add instructions here
 
 Add ansible playbook for post-os install
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  .. code::
-
-    cd /var/renasar
-    curl http://xfer.renasar.com/emc/ansible_test.tar.gz | tar xvzf -
+  TODO: update instructions here
 
 Configure the applications to start
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-All of the renasar programs are set to respect and verify the existance of a
-default configuration file. If this file doesn't exist, the services will
+All of the monorail programs are set to respect and verify the existence of a
+default configuration file (/opt/onrack/etc/monorail.json). If this file doesn't exist, the services will
```

Comment:
Yep - better to be accurate now - we've just been purging references to "onrack", but this one's baked in a bit right now...